docs/news/news-v5.0.x.rst

```diff
@@ -10,11 +10,11 @@ Open MPI version v5.0.1rc1
 
 - Internal PMIx and PRRTe versions:
 
-  - PMIx (v4.2.8rc1-1-g0e09e062). Commit hash: ``0e09e062e8cefe0feb18ea37d768ded213053d68``.
-  - PRRTE (v3.0.3rc1-2-g33aa50909f). Commit hash: ``33aa50909fe2c223f9f515fae120cd6a21d04653``.
- 
+  - PMIx (v4.2.8). Commit hash: ``d1c1ed0c2e64f19ad49291241a43630ea7fdce28``.
+  - PRRTE (v3.0.3). Commit hash: ``e383f5ad70c2633420d3425e9fb67e69b6bfd9c4``.
+
 - Bugfixes and changes (in chronological order)
-    
+
```

Comment:
Don't worry about this. We will update the doc again.

Reply:
For future releases, it would probably be good to automate this section of the doc (i.e., grab the specific git commits dynamically).  It would be just a little less for a human to need to do, and probably be less error-prone.